python/mxnet/ndarray/numpy/_op.py

```diff
@@ -900,6 +977,66 @@ def sqrt(x, out=None, **kwargs):
     return _unary_func_helper(x, _npi.sqrt, _np.sqrt, out=out, **kwargs)
 
 
+@set_module('mxnet.ndarray.numpy')
+def sign(x, out=None, **kwargs):
+    r"""
```

Comment:
remove `**kwargs` by override function signature

Reply:
Contents for rendering are from multiarray.py. All overridden already. But will try to fix _op.py. Thx!